src/diffusers/loaders.py

```diff
@@ -169,10 +180,11 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
                         user_agent=user_agent,
                     )
                     state_dict = safetensors.torch.load_file(model_file, device="cpu")
-                except EnvironmentError:
+                except IOError as e:
+                    if not allow_pickle:
+                        raise e
```

Comment:
@patrickvonplaten before you merge can you double check this error change is ok?